services/traction/api/main.py

```diff
@@ -1,10 +1,62 @@
 from fastapi import FastAPI
+from starlette.middleware import Middleware
+from starlette.middleware.base import (
+    BaseHTTPMiddleware,
+    RequestResponseEndpoint,
+)
+from starlette.requests import Request
+from starlette.responses import JSONResponse, Response
+from starlette_context import context, plugins
+from starlette_context.middleware import RawContextMiddleware
+
+from api.resources.ledger import router as ledger_router
 from api.resources.tenant import router as tenant_router
 
 from api.models import tenant  # noqa F401
+from api import acapy_utils as au
+
+
+
+class JWTTFetchingMiddleware(BaseHTTPMiddleware):
```

Comment:
Fast is built on Starlette, https://fastapi.tiangolo.com/features/#starlette-features. So i'm wondering if these should fastapi references instead of starlette directly.

Reply:
Possibly.  I'm just pulling in code from various examples at this point ...